source/grid/reference_cell.cc

```diff
@@ -75,8 +75,19 @@ namespace ReferenceCell
       }
     else if (reference_cell == Type::Tri)
       {
-        static const std::vector<Point<spacedim>> vertices = {
-          {{0.0, 0.0}, {1.0, 0.0}, {0.0, 1.0}}};
+        std::vector<Point<spacedim>> vertices;
```

Comment:
I know it's already merged, but wouldn't this be easier with 
```C++
std::vector<Point<spacedim>> vertices(3);
vertices[1][0] = 1.0;
vertices[2][1] = 1.0;
```
?

Reply:
Even simpler: #11617.